test/Microsoft.AspNetCore.Routing.Tests/Template/TemplateSegmentTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Routing.Patterns;
```

Comment:
Needs copyright header and sort (system first)

Reply:
Done